test/unit/Reflection/ReflectionMethodTest.php

```diff
@@ -334,21 +330,6 @@ public function testGetPrototype(string $class, string $method, ?string $expecte
         self::assertSame($expectedPrototype, $b->getDeclaringClass()->getName());
     }
 
-    public function testGetMethodNodeFailsWhenNodeIsNotClassMethod() : void
```

Comment:
Also here: no longer possible, as this change enforces the constraints via type hints.